scania-digital-design-system/tegel
# fix: pagination dropdown all rounded corners

## **Describe pull-request**  
Added rounded corners to pagination dropdown list.

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3593](https://tegel.atlassian.net/browse/CDEP-3593)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to amplify link.
2. Check in pagination rows-per-page dropdown, should now have all rounded corners (4px).
3. Check that dropdown component is not effected when used outside of table footer.

## **Checklist before submission**


[CDEP-3593]: https://tegel.atlassian.net/browse/CDEP-3593?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
